newIDE/app/src/ProjectManager/ExtensionsPropertiesDialog.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+import { Trans } from '@lingui/macro';
+
+import * as React from 'react';
+import FlatButton from '../UI/FlatButton';
+import Dialog from '../UI/Dialog';
+import Text from '../UI/Text';
+import { ColumnStackLayout } from '../UI/Layout';
+import EmptyMessage from '../UI/EmptyMessage';
+import PropertiesEditor from '../PropertiesEditor';
+import propertiesMapToSchema from '../PropertiesEditor/PropertiesMapToSchema';
+import { List } from '@material-ui/core';
+
+type Props = {|
+  project: gdProject,
+  open: boolean,
+  onApply: Function,
+|};
+
+class ExtensionsPropertiesDialog extends React.Component<Props> {
+  _onApply = () => {
+    this.props.onApply();
+  };
+
+  _mapExtensionValueToPropertyDescriptors(
+    properties: gdMapStringPropertyDescriptor,
+    extensionName: string,
+    project: gdProject
+  ) {
+    const keys = properties.keys().toJSArray();
+    for (let key of keys) {
+      properties
+        .get(key)
+        .setValue(
+          project.getExtensionPropertiesManager().getValue(extensionName, key)
+        );
+    }
+    return properties;
+  }
+
+  render() {
+    const { project } = this.props;
+    const allExtensions = project
+      .getCurrentPlatform()
+      .getAllPlatformExtensions();
+    const properties = [];
+    for (let i = 0; i < allExtensions.size(); i++) {
+      const extension = allExtensions.at(i);
+      const propertiesSchema = propertiesMapToSchema(
+        extension.getAllProperties(),
+        instance =>
+          this._mapExtensionValueToPropertyDescriptors(
+            extension.getAllProperties(),
+            extension.getName(),
+            project
+          ),
+        (instance, propertyName, newValue) =>
+          project
+            .getExtensionPropertiesManager()
+            .setValue(extension.getName(), propertyName, newValue)
+      );
+      if (
+        extension
+          .getAllProperties()
+          .keys()
+          .size() !== 0
+      )
+        properties.push(
+          <ColumnStackLayout key={extension.getName()}>
+            <Text size="title">
+              <Trans>{extension.getFullName()}</Trans>
+            </Text>
+            <PropertiesEditor
+              schema={propertiesSchema}
+              instances={[extension.getAllProperties()]}
```

Comment:
The instances must be `[project]`, not the properties. Instances is what is modified, which is here the project (the extension is not modified, extensions are just giving the property).